### PR TITLE
Prepare testing environment

### PR DESCRIPTION
**Pull request by Janix4000**
Include `Pytest` and create a directory `tests` with basic helper functions. In the future, we will generate `json` files with test elections and use them to ensure the correctness of our algorithms.

### CODE REVIEW

**Comment by Janix4000** [comment]
[Find a nice unit test library for python](https://trello.com/c/6EAxwOX9/16-find-a-nice-unit-test-library-for-python)

File: tests/test_bloc.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    committee_size = len(committee)
+    score = 0.0
+    for ballot in voting:
+        for candidate in committee:
+            score += 1.0 if ballot.pos(candidate) < committee_size else 0.0
+    return score
+
+
+def test_bloc():
+    print(os.getcwd())
```
Comment (Wolokin) [inline tests/test_bloc.py]: Should this be here?

File: tests/test_bloc.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    for voting, size_of_committee, expected_score in load_tests("tests/fixtures/bloc_medium.json"):
+        voting = OrdinalElection(voting)
+        committee = bloc(voting, size_of_committee)
+        score = scoring(voting, committee)
+        assert (score >= alpha * expected_score)
```
Comment (Wolokin) [inline tests/test_bloc.py:26]: Why not use assert_approximation here?

File: tests/util/util.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
+import json
+from typing import Callable, Iterable, Iterator
+
+
+def assert_approximation(committee: list, scoring_function, expected_score: float, alpha: float) -> None:
+    score = scoring_function(committee)
```
Comment (Wolokin) [inline tests/util/util.py]: I believe scoring_function should accept 'voting' as well

File: votepy/k-borda.py [hunk 4]
```diff
@@ -1,6 +1,8 @@
-from ordinal_election import OrdinalElection, OrdinalBallot
+from votepy.ordinal_election import OrdinalElection, OrdinalBallot
```
Comment (Wolokin) [inline votepy/k-borda.py]: We should rename this file with an underscore as well

**Review by Wolokin — CHANGES_REQUESTED** [review]

File: tests/util/util.py [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    # Args:
+        `filepath` (str): File path of newly generated test file
+        `elections` (Iterable[tuple[Candidates, Voters]]): Collections of (n_candidates, n_voters) pairs, describing elections for generator
+        `generator` (Callable[[Candidates, Voters], Election]): Function generating elections based on (n_candidates, n_voters) arguments
+        `solver` (Callable[[Election, CommitteeSize], Score]): Function returning the score of the committee, for the given election
+    """
+    all_tests = []
+    for candidates_size, voters_size in elections:
+        election = generator(candidates_size, voters_size)
+        test = {}
```
Comment (msluszniak) [inline tests/util/util.py]:
```suggestion
        test = {"election": election, "scores": []}
```

File: votepy/bloc.py [hunk 6]
```diff
@@ -1,54 +1,70 @@
...
+
+    ### Args:
+        `voting` (OrdinalElection | list): voting for which the function calculates the committee
+        `size_of_committee` (int): Size of the committee
 
-    Args:
-        voting (OrdinalElection | list): voting for which the function calculates the committee
-        size_of_committee (int): Size of the committee
+    ### Raises:
+        `ValueError`: Size of committee is a positive number which do not exceeds number of all candidates
```
Comment (msluszniak) [inline votepy/bloc.py]:
```suggestion
        `ValueError`: The size of the committee is a positive integer which does not exceed the number of all candidates
```

File: votepy/bloc.py [hunk 7]
```diff
@@ -1,54 +1,70 @@
...
+        `ValueError`: Number of candidates scored in Bloc is a positive number which do not exceeds number of all candidates
```
Comment (msluszniak) [inline votepy/bloc.py]:
```suggestion
        `ValueError`: The number of candidates scored in Bloc is a positive integer which does not exceed the number of all candidates
```

File: tests/test_bloc.py (hunk 1 above)
Comment (Janix4000) [reply tests/test_bloc.py]: No

File: tests/test_bloc.py (hunk 2 above)
Comment (Janix4000) [reply tests/test_bloc.py:26]: To be honest, `assert_approximation` is unnecessary and should be removed.
In many cases it is easier to create this assert by hand.

File: tests/util/util.py (hunk 3 above)
Comment (Janix4000) [reply tests/util/util.py]: I removed this function.

**Review by msluszniak — APPROVED** [review]

**Review by Wolokin — APPROVED** [review]

**Pull request merged**